BraintreePayPal/PayPalOneTouch/Analytics/PPFPTITracker.h

```diff
@@ -33,7 +33,7 @@
 
 
 /// The delegate which actually sends the data
-@property (nonatomic, weak, readwrite) id<PPFPTINetworkAdapterDelegate> networkAdapterDelegate;
+@property (nonatomic, weak, readwrite) _Nullable id<PPFPTINetworkAdapterDelegate> networkAdapterDelegate;
```

Comment:
Thanks for the fix! Do you mind if we move the nullability annotation into the parentheses? It's just a style suggestion.

```
+@property (nonatomic, weak, readwrite, nullable) id<PPFPTINetworkAdapterDelegate> networkAdapterDelegate;
```